src/rpcmining.cpp

```diff
@@ -45,9 +45,13 @@ Value GetNetworkHashPS(int lookup, int height) {
     if (pb == NULL || !pb->nHeight)
         return 0;
 
-    // If lookup is -1, then use blocks since last difficulty change.
+    // If lookup is -1, then use difficulty averaging window.
```

Comment:
I left it consistent with upstream, but can change (won't be for eight hours though).